resources/odd/source/frus.odd

```diff
@@ -192,11 +192,13 @@
                     </model>
                     <model behaviour="list" predicate="ancestor::div[@xml:id='persons']" cssClass="list-person">
                         <param name="content">item</param>
-                        <outputRendition>list-style-type: none; margin-left: 2em; text-indent: -2em; padding-left: 0</outputRendition>
                     </model>
-                    <model behaviour="list" cssClass="list">
+                    <model behaviour="list" cssClass="hsg-list">
+                        <param name="content">item</param>
+                    </model>
+                    <model behaviour="list" predicate="ancestor::list" cssClass="hsg-nested-list">
```

Comment:
@plutonik-a this model would never be applied, you need to move it above the previous one, since the way ODD is processed assumes only the first rule that matches (in the document order in ODD) is fired